src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1686,9 +1697,9 @@ impl A320DoorController {
                 self.control_position_request = if self.position_requested > Ratio::new::<ratio>(0.)
                     || self.duration_in_hyd_control < Self::UP_CONTROL_TIME_BEFORE_DOWN_CONTROL
                 {
-                    Ratio::new::<ratio>(1.)
+                    Ratio::new::<ratio>(1.0)
                 } else {
-                    Ratio::new::<ratio>(0.)
+                    Ratio::new::<ratio>(-0.1)
```

Comment:
What is the reason for this being initialized at `-0.1` instead of `0`?

Reply:
First PR only implemented ON/OFF position control, meaning full force until position reached.
Now I use a general formula that is doing position control at requested position.  So for cargo door for example, I expect a ON/OFF behaviour, and don't want the actuator to try to exactly match close position. This is ensured by asking a position further than close position.
Will add documentation about this somewhere......